rpcd/playbooks/roles/rpc_maas/templates/filebeat_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "filebeat_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : process_check.py
+    args    : [ {% for name in filebeat_process_names %}{{ name }}{% if not loop.last %}, {% endif %}{% endfor %}, {%if "{{ ansible_ssh_host }}" in groups["all_containers"]%} -c "{{ ansible_ssh_host }}" {% else %}{% endif %}]
```

Comment:
{% if "{{ ansible_ssh_host }}" in groups["all_containers"] %} will never evaluate to true.
ansible_ssh_host is the IP of the host, which will never match a host in all_containers. Maybe you meant to say inventory_hostname, here?
Unless I'm missing something about the way Ansible inspects these groups?